experimental/GModule/GModule.jl

```diff
@@ -299,10 +263,6 @@ function invariant_lattice_classes(M::GModule{<:Oscar.GAPGroup, <:AbstractAlgebr
   return invariant_lattice_classes(MZ)
 end
 
-function Oscar.pseudo_inv(h::Generic.ModuleHomomorphism)
-  return MapFromFunc(codomain(h), domain(h), x->preimage(h, x))
-end
-
 function _hom(f::Map{<:AbstractAlgebra.FPModule{T}, <:AbstractAlgebra.FPModule{T}}) where T
```

Comment:
is this still used? It feels like either it should move or be deleted

Reply:
`_hom` is used just once, a few lines more down, in order to abbreviate the code.
If we move it, we should call it `hom` (?), and document what it is expected to do:
Take a map, assume that this map is linear, and return the corresponding "linear-map-by-images".
(In the current situation, I think it would be better to do this directly, by mapping generators under the various maps, instead of first creating several intermediate maps; but this would look less elegant.)
In this sense, deleting `_hom` is perhaps a good idea.